.storybook/main.js

```diff
@@ -7,4 +7,27 @@ module.exports = {
     "@storybook/addon-react-native-web",
   ],
   framework: "@storybook/react",
+  webpackFinal: async config => {
```

Comment:
**Praise**: 👏🏻 Thank you for this. Must have missed it when setting up storybook